frontend/app/pages/confirm.vue

```diff
@@ -0,0 +1,17 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <div class="flex flex-col md:flex-row justify-center items-center bg-gray-100 dark:bg-zinc-900 rendering-pixelated bg-[url(assets/img/background.webp)] bg-cover bg-center">
```

Comment:
I think just toast could be better and simpler. But if you want to keep it, create compinent firbcentered div with backgriynd

Reply:
You have to add patrameter and parse it in nuxt (remove it sndand display)